voluncheer/map/models.py

```diff
@@ -1,3 +1,18 @@
-# from django.db import models
+from django.db import models
+
 
 # Create your models here.
+class NYCharities(models.Model):
+    class Meta:
+        verbose_name_plural = "NYCharities"
```

Comment:
Are these all across NY or just in NYC? If the latter do you think we should have two "C"s in the name? NYCCharities?

Reply:
Visually, I like the one C better, but if we are being very specific I can change the name.

Reply:
Nah, that's fine. Just wanted to make sure we were thinking about it.